src/components/DocumentWithEvaluations/components/EmptyEvaluationsView.tsx

```diff
@@ -0,0 +1,90 @@
+"use client";
+
+import { useRef } from "react";
+import Link from "next/link";
+import SlateEditor from "@/components/SlateEditor";
+import { ArrowTopRightOnSquareIcon, InformationCircleIcon } from "@heroicons/react/20/solid";
+import type { Document } from "@/types/documentSchema";
+
+interface EmptyEvaluationsViewProps {
+  document: Document;
+  contentWithMetadataPrepend: string;
+}
+
+export function EmptyEvaluationsView({
+  document,
+  contentWithMetadataPrepend,
+}: EmptyEvaluationsViewProps) {
+  const contentRef = useRef<HTMLDivElement>(null);
+
+  return (
+    <div className="flex h-full flex-col overflow-x-hidden">
+      {/* Message banner at the top */}
+      <div className="mx-5 mt-3 rounded-lg border border-blue-200 bg-blue-50 p-4">
+        <div className="flex items-center gap-2">
+          <InformationCircleIcon className="h-5 w-5 text-blue-600" />
+          <p className="text-sm text-blue-800">
+            This document hasn't been evaluated yet. You can still read the content below.
```

Comment:
The text should vary based on the viewer. 

If it's the document owner, this should read, "This document has no evaluations. Add some [here]".  with a link to /docs/${document.id}`. 

If it's not the document owner, it should just say, "This document doesn't have any evaluations, but you can still read the content below."